app/src/main/java/nextstep/github/ui/screens/list/components/RepositoryItem.kt

```diff
@@ -0,0 +1,59 @@
+package nextstep.github.ui.screens.list.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import nextstep.github.model.Repository
+import nextstep.github.ui.theme.GithubTheme
+
+@Composable
+fun RepositoryItem(
+    repository: Repository,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+    ) {
+        Text(
+            text = repository.fullName,
+            style = MaterialTheme.typography.titleLarge,
+        )
+        Text(
+            text = repository.description ?: "",
```

Comment:
.orEmpty() 확장 함수를 활용해볼 수도 있겠어요 :)
```suggestion
        Text(
            text = repository.description.orEmpty(),
```

Reply:
아~ 이런게 있었군요! 훨씬 깔끔하네요 ㅋㅋㅋㅋ